### DISCUSSION TITLE
CarPlay - Show More Info on Play Screen

### DISCUSSION THREAD

**Discussion by mandopatriot** [LeoKlaus/plappa#52] [Feature Requests]
During CarPlay using the audiobook play screen, please add more information such as author, narrator, etc. Currently the only items that show are chapter number, book name, book picture/icon, play/pause, rewind/fast forward, and timestamp menu. The play screen on bigger car screens is sparse, a little more information could be useful or look better. This is with ABS 2.3.1 docker as server, iOS 17.3.1 on iPhone 15 Pro.

**Comment by LeoKlaus** [comment]
You mean the "Now playing" screen in CarPlay?
<img width="512" alt="image" src="https://github.com/LeoKlaus/plappa/assets/66492595/65526763-1922-4189-a250-30aae5313a6d">
What information would you like to see there?

CarPlay works with "templates", so even as a developer you only have very little choice which elements to put where, but I'll surely see what's possible!

**Reply by mandopatriot** [reply]
Apologies, I don’t have anything else specific outside of the author and maybe the narrator. I know more is possible because I previously used Prologue with Plex, but I’ve already remove my Plex setup so I can’t verify what it shows. Just my recollection is that CarPlay for Plappa vs Prologue, Plappa seems to show less details.

**Comment by LeoKlaus** [comment]
I've added some additional information with one of the recent updates (1.1.3?), so I'll close this for now.

If there's anything specific you're missing, feel free to reopen.
